### ISSUE TITLE
Mind Grind install errors

### ISSUE THREAD

**Issue by weirddan455** [dosbox-staging/dosbox-staging#3703]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Linux x86_64

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

I am getting the following errors on install.  Install can be completed by clicking Retry on the first error and Ignore on the 2nd.

![image0005](https://github.com/dosbox-staging/dosbox-staging/assets/8128047/33b59873-7043-4842-a7d7-b6733f7faef1)

![image0006](https://github.com/dosbox-staging/dosbox-staging/assets/8128047/bf24f12f-8ac6-43dd-8e58-7b9133262064)

### Steps to reproduce the behaviour.

Explain how to reproduce
1.  Mount the ISO
2.  Start Windows 3.1
3.  Run setup.exe

### Download URL of affected game or software

https://archive.org/details/mind-grind

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by weirddan455** [comment]
I'm also getting the following error attempting to start the game.  I may have messed up the Windows sound drivers somehow though.  The Creative tools that got installed are throwing .DLL errors.

![image0007](https://github.com/dosbox-staging/dosbox-staging/assets/8128047/52b5f775-0afc-4cdc-b003-59c47af73853)

**Comment by Grounded0** [comment]
Does this game run properly? I'm asking since we're dropping https://github.com/dosbox-staging/dosbox-staging/labels/game%20compatibility for installer related issues if the game itself run properly.

**Comment by Grounded0** [comment]
I tried to install and run this game and seems to have no issues after 10 minutes of play. I do get that can't overwrite `TOOLHELP.DLL` error, but that's due to the DLL being already loaded on Windows boot up as shown in Microsoft System Info diagnostic tool.

Here's information about `TOOLHELP.DLL`:

https://jacobfilipp.com/DrDobbs/articles/DDJ/1992/9209/9209i/9209i.htm

My suggestion is that this game runs on 0.82.2 with the drivers I have without issues including in-game help and this ticket can be closed as completed:

<img width="1392" height="1100" alt="Image" src="https://github.com/user-attachments/assets/19d83590-cf0c-4921-bc31-70afb69b5711" />

<img width="1392" height="1100" alt="Image" src="https://github.com/user-attachments/assets/a5e583f3-a6af-46f0-a63a-6fb354c3bd0f" />

**Comment by Grounded0** [comment]
@johnnovak 

Have I collected enough data to close this in the above post?

**Comment by johnnovak** [comment]
> [@johnnovak](https://github.com/johnnovak)
> 
> Have I collected enough data to close this in the above post?

Dropping the game compatibility label, but we wanna keep this as the installer issue points to an emulation problem.

**Comment by Grounded0** [comment]
There isn't an installer problem. The installer tries to overwrite a DLL that's loaded on Windows boot up. That's a core Windows library that the installer tries to overwrite. Badly configured installers aren't our issue.

`TOOLHELP.DLL` functionality was taken over mainly by `kernel32.dll` in Windows 95, so the installer would run without issues on Windows 95. Maybe the dev team didn't test how the installer would run on Windows 3.x or presumed users would just select ignore to allow the installer to complete.

**Comment by johnnovak** [comment]
@Grounded0 Thanks for the extra explanation, I wasn't aware of these details.

@weirddan455 Are you happy to close this given @Grounded0 's reasoning? It seems this is just a badly coded game, after all. I could test the installer on my MMX box to confirm the real hardware behaviour.

**Comment by weirddan455** [comment]
Yeah, I really don't remember the details of this other than what I posted because it was over a year ago. I also rarely do any Windows 3.1 stuff.

I think this was around the time I implemented file sharing and this was one of the games I was testing so I filed this in case in happened to be related. It's probably not though so I'm fine to close this (or keep it open until you test on real hardware if you want).

**Comment by johnnovak** [comment]
> It's probably not though so I'm fine to close this (or keep it open until you test on real hardware if you want).

I'll test on hardware, just to get the definitive answer.
